connect/src/com/telenor/connect/ui/ConnectWebViewClient.java

```diff
@@ -185,9 +186,12 @@ public WebResourceResponse fetchUrlTroughCellular(String originalUrl) {
         return null;
     }
 
+    @RequiresApi(api = Build.VERSION_CODES.M)
```

Comment:
Err, why is Marshmallow required for this functionality? What happens to all of the users on older systems?

Reply:
I see, thanks.